templates/security/fips.html

```diff
@@ -0,0 +1,209 @@
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3622">#3622</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List">Under validation</a>
+          </td>
+        </tr>
+        <tr>
+          <th>libgcrypt</th>
+          <td class="u-align--center" aria-label="None">–</td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3748">#3748</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3902">#3902</a>
+          </td>
+        </tr>
+        <tr>
+          <th>StrongSwan</th>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2978">#2978</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3648">#3648</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List">Under validation</a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140-2 validated modules</a>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h3>Canonical patches vulnerabilities</h3>
+    <p>Each FIPS 140-2 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/fips">our security pages on how to enable these options</a>.</p>
+    <p>We recommend enabling the 'fips-updates' option that includes the security fixes. Our cloud images starting with 20.04 are pre-configured to enable security updates. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance phase</a>.</p>
```

Comment:
```suggestion
    <p>We recommend enabling the 'fips-updates' option that includes the security fixes. Our cloud images starting with 20.04 are pre-configured to enable security updates. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance phase</a>.</p>
```
Change diagonal apostrophes to straight ones